src/components/product/IFXProductCreateEdit.vue

```diff
@@ -207,8 +216,17 @@ export default {
                         persistent-hint
                       ></v-text-field>
                     </v-col>
+                    <v-col cols="12" md="4">
+                      <v-text-field
+                        v-model="item.description"
+                        label="Description"
+                        data-cy="rate-description"
+                        :error-messages="errors.description"
+                        persistent-hint
+                      ></v-text-field>
+                    </v-col>
                     <v-col>
-                      <v-switch v-model="item.active" label="Active" data-cy="rate-active"></v-switch>
+                      <v-switch v-model="item.active" :disabled="true" label="Active" data-cy="rate-active"></v-switch>
```

Comment:
Rather than disable it, should we just show the word "Active" for now? If it's disabled, people often wonder what they have to do to enable it.